infra/modular-examples/aws-google-workspace/variables.tf

```
@@ -81,12 +81,17 @@ variable "connector_display_name_suffix" {
 
 variable "psoxy_base_dir" {
   type        = string
-  description = "the path where your psoxy repo resides. Preferably a full path, /home/user/repos/, avoid tilde (~) shortcut to $HOME"
+  description = "the path where your psoxy repo resides"
+  default     = null # will use `"${path.root}/.terraform/"` if not provided
 
   validation {
-    condition     = can(regex(".*\\/$", var.psoxy_base_dir))
+    condition     = var.psoxy_base_dir == null || can(regex(".*\\/$", var.psoxy_base_dir))
     error_message = "The psoxy_base_dir value should end with a slash."
   }
+  validation {
```

Empty line between two validations? Same for other files:

```suggestion

  validation {
```